tests/augmentation/test_container.py

```diff
@@ -672,6 +673,39 @@ def test_inverse_and_forward_return_transform(self, random_apply, device, dtype)
         if random_apply is False:
             reproducibility_test((inp, mask, bbox, keypoints, bbox_2, bbox_wh, bbox_wh_2), aug)
 
+    def test_transform_list_of_masks_and_boxes(self, device, dtype):
+        input = torch.randn(2, 3, 256, 256, device=device, dtype=dtype)
+        mask = [
+            torch.ones(1, 3, 256, 256, device=device, dtype=dtype),
```

Comment:
which failed randomly before, is sometimes, one of the augmentations complains about the masks be not batched... but it looks to be fine with a list of batched masks... 

@shijianjian Do you think it is fine to go this way? Looks like it partially adds the support to list of masks

Reply:
Great. LGTM.

Reply:
BTW, maybe not in this PR. I think we also wanted to support instance segmentation. So, we will need something similar to bboxes later on.

Reply:
I don't get what you mean, because from what I see, this patch already supports a list of bbox